alerting/build.gradle

```diff
@@ -3,7 +3,6 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import com.github.jengelman.gradle.plugins.shadow.ShadowPlugin
```

Comment:
In my check on JS plugin, gradleup did not remove this `com.github.jengelman.gradle.plugins.shadow.ShadowPlugin`, just the namespace of the plugin on gradle mirror is changed.

I just update the `apply` section and it still pass the assemble.

Reply:
Ah thanks for that context! I'll try that.